SECURITY.md

```diff
@@ -1,4 +1,6 @@
+# Security
```

Comment:
is this required by the linter? otherwise [we don't actually __need__ it](https://github.com/OpenZeppelin/cairo-contracts/security/policy)

Reply:
Yeah, having a heading in .md files is required by the linter. We could have it ignore SECURITY.md or ignore the heading rule if we want to keep security as is